README.md

```diff
@@ -29,80 +31,117 @@ docker-compose up deps
+By default, the verification of the Root Of Trust is disabled. In order to enable it set the env variable '
+VERIFY_ROOT_OF_TRUST=True'.
+The default root of trust is the GLEIF
+external(https://github.com/WebOfTrust/WebOfTrust.github.io/blob/main/.well-known/keri/oobi/EDP1vHcw_wc4M__Fj53-cJaBnZZASd-aMTaSyWEQ-PC2/index.json)
+If you are running Keria locally, you will need to add your own Root Of Trust. To do that you need to set the env
+variable 'VERIFIER_ENV=development'. And then
+send the POST request to the localhost:7676/root_of_trust/{AID-of-the-root-of-trust} with application/json+cesr format
+data containing the CESR of the new Root Of Trust.
 
 # Configuration:
+
 ## Running the Verifier Service
 
 To run the Verifier service, you need to specify a configuration file by providing the following arguments:  
 `--config-dir=scripts --config-file='your_config_file'`
 
 ---
 
-## Configuration Files Location
+## Configuration Files
 
-Configuration files are located in the following directory:  
-`scripts/keri/cf`
-
----
+The Verifier uses configuration files to define trusted LEIs, allowed roles and schemas. These files are located in:  
+**`scripts/keri/cf`**
 
 ## Configurable Parameters
 
-You can customize the service behavior using these configuration options:  
+You can customize the service behavior using these configuration options:
 
-- **`iurls`**: - OOBI URLs
+- **`iurls`**: - Witnesses OOBI URLs
 - **`durls`**: - Schema OOBI URLs
-- **`trustedLeis`**: A list of trusted LE identifiers.  
-- **`allowedEcrRoles`**: Roles permitted for ECR credential authorization.  
-- **`allowedOorRoles`**: Roles permitted for OOR credential authorization.  
+- **`trustedLeis`**: A list of trusted LE identifiers.
+- **`allowedEcrRoles`**: Roles permitted for ECR credential authorization.
+- **`allowedOorRoles`**: Roles permitted for OOR credential authorization.
 - **`allowedSchemas`**: A list of schemas allowed for authorization.
 
+### Default Configuration
+The default configuration file, **`verifier-config-public.json`**, is located in the **`scripts/keri/cf`** directory. By default:  
+- **`trustedLeis`**, **`allowedEcrRoles`**, and **`allowedSchemas`** are empty.  
+  - This means Schema and Role checks will fail, and any credential authorization will be rejected.  
+- You must populate **`allowedEcrRoles`** and **`allowedSchemas`** or use a pre-configured file from **`scripts/keri/cf/examples`**.  
+- You can also use **`verifier-config-test.json`** which has values set for the allowedEcrRoles and allowedSchemas.
+
+### Root of Trust Configuration
+The default root of trust points to **production Gleif External**:  
+`EINmHd5g7iV-UldkkkKyBIH052bIyxZNBn9pq-zNrYoS`
+
+If your credentials are not chained to this production root of trust, you must:  
+
+Modify the file **`src/root_of_trust_oobis/gleif_external.json` to make the root of trust match the one your credentials are chained to.**
```

Comment:
We only recommend changing gleif_external.json if Gleif External changed. But this reads as if you would change gleif_external.json even when you wanted to use some other root of trust (such as a test root of trust).